content/source/docs/cloud/overview.html.md

```diff
@@ -44,7 +43,7 @@ Remote execution helps provide consistency and visibility for critical provision
 
 [execution_mode]: ./workspaces/settings.html#execution-mode
```

Comment:
this is a nit and out of scope, but can we replace the bullets with a numbered list? we introduce it by saying there are "three workflows"

![image](https://user-images.githubusercontent.com/7556325/144981915-aceefb18-b26c-4b89-a164-828db6f0ba6c.png)

Reply:
hmmmm I see what you're saying, but I think we want to reserve numbered lists for either steps that must be done in order OR for things that have a clear precedence. I'm going to keep this as bulleted because here all three workflows are valid.